pkg/api/defaults_test.go

```diff
@@ -772,6 +772,7 @@ func TestContainerRuntime(t *testing.T) {
 	mockCS := getMockBaseContainerService("1.10.13")
 	properties := mockCS.Properties
 	properties.OrchestratorProfile.OrchestratorType = Kubernetes
+	properties.OrchestratorProfile.KubernetesConfig.MobyVersion = "3.0.4"
```

Comment:
Can we do a for loop with at least two < 3.0.5 versions as input? Just to make sure that we don't accidentally regress the test if the static string actually matches the value of `DefaultMobyVersion`